infinite_amp.theme

```diff
@@ -108,8 +108,10 @@ function infinite_amp_preprocess_taxonomy_term(&$variables) {
   if (isset($variables['content']['field_paragraphs'])) {
     foreach (Drupal\Core\Render\Element::children($variables['content']['field_paragraphs']) as $key) {
       $paragraph = $variables['content']['field_paragraphs'][$key]['#paragraph'];
-      if (in_array($paragraph->bundle(), ['content_teaser', 'recent_content'])) {
-        $variables['content']['field_paragraphs'][$key] = ['#markup' => ''];
+      if ($paragraph) {
```

Comment:
To make the code more readable we should avoid multiple nested if / for statements. Instead of  
`if($paragraph) {}`  
you could use
```
if (null === $paragraph) {
  continue;
}
```
That way we can remove one level of nesting